dev-client/src/screens/LocationScreens/LocationDashboardContent.tsx

```diff
@@ -129,11 +130,13 @@ export const LocationDashboardContent = ({
           <Row>
             <RadioBlock
               label={
-                <Row>
+                <Row alignItems="center">
```

Comment:
Design nit: This was the case before your PR, but the vertical gap between the privacy line and the line above, is slightly larger than the gap between all other lines. This PR just makes it very slightly more pronounced. Wanted to mention it but happy for it to not matter.
![Screenshot 2024-08-15 at 6 11 40 PM](https://github.com/user-attachments/assets/921099b4-6bfb-42da-8fdf-4ea5cc675d36)

Reply:
Will review with Courtney today - not sure if there's anything we can do about it if the icon results in a taller line but I noticed it too.

Reply:
Confirmed that this is acceptable.